### ISSUE TITLE
Installing the latest version

### ISSUE THREAD

**Issue by dottodot** [flynn/flynn#2766]
The version that's getting installed seemed to be quite old i.e v20160327.0 which is over a month ago.

If there a way to install a more up to date version or is that a bad idea?

**Comment by lmars** [comment]
@dottodot we have two release channels: stable and nightly.

We release the nightly channel every day (provided the `master` branch has passed CI), and we currently release to the stable channel manually when we feel a nightly version has been tested "in the real world".

By default, the install script will use the stable channel (which is currently pointing at `v20160327.0`) but you can instead use the nightly channel by setting `FLYNN_UPDATE_CHANNEL=nightly` when running the install script:

```
sudo FLYNN_UPDATE_CHANNEL=nightly bash < <(curl -fsSL https://dl.flynn.io/install-flynn)
```

See [the Stability doc](https://flynn.io/docs/stability) for more information.

**Comment by philiplb** [comment]
Can I switch to nightly and later back to stable?

**Comment by lmars** [comment]
@philiplb in theory yes, but it is not a well tested approach (switching from nightly back to stable and then running `flynn update` could try and revert to an older version, something we do not currently make effort to support).

Feel free to try it though, the current channel is stored in `/etc/flynn/channel.txt`, so to switch to nightly run `echo nightly > /etc/flynn/channel.txt` as root.

**Comment by philiplb** [comment]
@lmars Gotcha, thank you.

**Comment by dottodot** [comment]
So when I see on your blog that there has been bug fixes I'm assuming that's only being applied to the nightly build.

**Comment by lmars** [comment]
@dottodot once a bug fix is merged into master, it will make it into a nightly release (provided master has passed CI) and we also blog that the change has been merged, but we tend to only release those fixes to the stable branch once we have verified there are no regressions "in the real world" for that release, which is currently a decision we make manually and at varying frequency.
